pkg/networkpolicy/multinetegressrule.go

```diff
@@ -129,3 +243,27 @@ func (builder *EgressRuleBuilder) GetEgressRuleCfg() (*v1beta1.MultiNetworkPolic
 
 	return builder.definition, nil
 }
+
+func (builder *EgressRuleBuilder) validate() (bool, error) {
+	objectName := "MultiNetworkPolicyEgressRule"
```

Comment:
```suggestion
	objectName := "multiNetworkPolicyEgressRule"
```
since this ends up starting an error message it should start with a lowercase letter

Reply:
done